test/heat-images/heat_tester.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from textwrap import dedent
+
+from grpc4bmi.bmi_client_docker import BmiClientDocker
+from grpc4bmi.bmi_optionaldest import OptionalDestBmi
+
+def test_heat(tmp_path: Path):
+    # Config file for Python based heat model
+    py_config = dedent("""\
+        shape:
+        - 6
+        - 8
+        spacing:
+        - 1.0
+        - 1.0
+        origin:
+        - 0.0
+        - 0.0
+        alpha: 1.0
+    """)
+    # Config file for C/C++ based heat model
+    c_config = '1.5, 8.0, 7, 6'
+
+
+    image, config_body = (
+        # Comment out line you want to run
+    'heat:py-0.2',py_config
```

Comment:
Fixed in #129

Reply:
I will have a look at the division by zero as the workaround in bmi_grpc_client.py::get_var_itemsize should fix it

Reply:
That division by zero workaround is only in branch of this PR at https://github.com/eWaterCycle/grpc4bmi/blob/latest-protobuf/grpc4bmi/bmi_grpc_client.py#L199 so that makes sense then when running the bmi2 branch it fails.